src/generic_ui/polymer/root.html

```diff
@@ -140,6 +140,26 @@
         top: 20%;
         z-index: 1002; /* Must be greater than core-overlay-backdrop */
       }
+      paper-toast {
+        position: fixed;
+        left: 12px;
+        right: 12px;
+        background-color: #283230;
+        padding: 16px 16px 12px 20px;
+        font-size: 12px;
+      }
+      #toastHelpLink {
+        color: #47B5AA;
+      }
+      core-icon[icon="launch"] {
```

Comment:
I'm not seeing this icon in root.html - is it still used?  If so, can you change this rule to maybe ```paper-toast core-icon[icon="launch"]``` so it's clear it's part of the toast?